api/node/README.md

```diff
@@ -33,7 +33,7 @@ You need to install the following components:
 
   * **[Node.js](https://nodejs.org/download/release/)** (v16. or newer)
   * **[pnpm](https://www.pnpm.io/)**
-  * **[Rust compiler](https://www.rust-lang.org/tools/install)** (1.82 or newer)
```

Comment:
Removed MSRV in the node docs because for a long time, Rust has been good at showing that the version is outdated, and the link two line bellow already mention the MSRV. People shouldn't build Slint from source anyway. So that's less things to update when changing MSRV.

Reply:
Thanks, good point